samples/ComplianceServer/ComplianceServer.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Rather than this being a sample, can we instead structure it in a way where it's a test suite that gets run as part of our tests?